src/Notification.php

```diff
@@ -27,6 +27,13 @@
 
 class Notification
 {
+    const PROPERTY_EVENT_CODE = 'eventCode';
+    const PROPERTY_SUCCESS = 'success';
+
+    const REQUIRED_DATA = [
+        self::PROPERTY_EVENT_CODE
+    ];
+
```

Comment:
How about adding `success` to `REQUIRED_DATA` except when event code is `REFUND_FAILED`?

Reply:
Updated this in https://github.com/Adyen/php-webhook-module/pull/3/commits/3542a30c9d7917ff347700526ebb824e99b2da89 since upon further investigation in the [docs](https://docs.adyen.com/api-explorer/#/Webhooks/v1/post/REFUND_FAILED) this notification also includes the `success` field.